packages/create-aragon-app/src/util.js

```diff
@@ -1,13 +1,21 @@
+const commandExistsSync = require('command-exists').sync
 const execa = require('execa')
+const fs = require('fs-extra')
+const path = require('path')
 
 const PGK_MANAGER_BIN_NPM = 'npm'
+const PGK_MANAGER_BIN_YARN = 'yarn'
 
-const getNodePackageManager = () => {
-  return PGK_MANAGER_BIN_NPM
-}
+const getNodePackageManager = useYarn =>
+  useYarn ? PGK_MANAGER_BIN_YARN : PGK_MANAGER_BIN_NPM
+
+const installDeps = (oldTemplate, cwd, task) => {
+  const useYarn = commandExistsSync('yarn') && !oldTemplate
+
+  // If we don use yarn we delete yarn.lock
+  if (!useYarn) fs.removeSync(path.join(cwd, 'yarn.lock'))
```

Comment:
I think there's also the `app/yarn.lock` file.

Reply:
Actually, I would perhaps keep the `yarn.lock` files. It's not necessarily bad to support both yarn and npm.